SpiceSharp/Components/Subcircuits/Parameters/InitialConditionParameters.cs

```diff
@@ -0,0 +1,21 @@
+﻿using SpiceSharp.Attributes;
+using SpiceSharp.Behaviors;
+
+namespace SpiceSharp.Components.SubcircuitBehaviors
+{
+    /// <summary>
+    /// Parameters for <see cref="IInitialConditionBehavior"/>
+    /// </summary>
+    /// <seealso cref="ParameterSet" />
+    public class InitialConditionParameters : ParameterSet
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether initial conditions are applied in parallel.
+        /// </summary>
+        /// <value>
+        ///   <c>true</c> if initial conditions are applied in parallel; otherwise, <c>false</c>.
+        /// </value>
+        [ParameterName("ic.ic"), ParameterInfo("Flag indicating whether initial conditions are applied in parallel")]
```

Comment:
ic.ic ?

Reply:
I know, but it _is_ consistent with other subcircuit parameters...